form-flow/src/test/resources/config/form-flow/inkomens_loket.json

```diff
@@ -0,0 +1,62 @@
+{
+    "key": "inkomens_loket",
+    "startStep": "woonplaats",
+    "steps": [
+        {
+            "key": "woonplaats",
+            "nextSteps": [
+                {
+                    "condition": "${submissionData.woonplaats.inUtrecht == true}",
+                    "step": "leeftijd"
+                },
+                {
+                    "condition": null,
+                    "step": "end"
+                }
+            ]
+        },
+
+        {
+            "key": "leeftijd",
+            "nextSteps": [
+                {
+                    "condition": "${submissionData.leeftijd.isOuderDan21 == true && submissionData.leeftijd.isJongerDanAOW == true}",
+                    "step": "gezinssituatie"
+                },
+                {
+                    "step": "end"
```

Comment:
Not sure if it's on purpose, but i like how the condition is missing here, and set explicitly to null in the step above. This way both configuration methods are tested.

Reply:
Yes, was on purpose